lighthouse-core/gather/gather-runner.js

```diff
@@ -91,13 +91,14 @@ class GatherRunner {
 
   static setupDriver(driver, options) {
     log.log('status', 'Initializing…');
+    const resetStorage = !(options.flags && options.flags.disableStorageReset);
```

Comment:
is `options.flags` ever not defined here? If so, the `options.flags.blockedUrlPatterns` line below is in trouble